parm/config/config.atmensanal

```diff
@@ -0,0 +1,27 @@
+#!/bin/ksh -x
+
+########## config.atmensanal ##########
+# configuration common to all atm atmensanal analysis tasks
+
+echo "BEGIN: config.atmensanal"
+
+export OBS_YAML_DIR=$HOMEgfs/sorc/gdas.cd/parm/atm/obs/config/
+export OBS_LIST=$HOMEgfs/sorc/gdas.cd/parm/atm/obs/lists/lgetkf_prototype.yaml
+export BERROR_YAML=$HOMEgfs/sorc/gdas.cd/parm/atm/berror/hybvar_bump.yaml
+export ATMENSYAML=$HOMEgfs/sorc/gdas.cd/parm/atm/lgetkf/lgetkf.yaml
+##export FV3JEDI_FIX=/work2/noaa/da/cmartin/GDASApp/fix
+##export FV3JEDI_FIX=/work2/noaa/da/fhan/GDASApp/fix
+export FV3JEDI_FIX=/work2/noaa/da/rtreadon/GDASApp/fix
```

Comment:
New fix files should go into the main fix sets. @RussTreadon-NOAA please provide the best pickup location for the new JEDI fix files and let me know the preferred subfolder name for them (`fix_jedi`?). I can pull them into the main set and then rsync to the other platform copies. Go-live on WCOSS2 is tomorrow so this may take me a day or so to get to. Thanks!

Reply:
Since this will be an iterative process, I can assume (for the time being) the responsibility for maintaining a `fix_jedi` directory in the main g-w fix on Orion, Hera, and WCOSS2.   Yes, we cannot yet run JEDI on WCOSS2 but I hope we can do so before we press to far into fall 2022.   I'll add `fv3_jedi` to `sorc/link_workflow.sh`.   JEDI config files will then be updated to point at `fix_jedi` in the g-w clone.  Does this work for everyone?

Reply:
@RussTreadon-NOAA Yes, Kate and I agree this is the best path for now. When fix_jedi becomes more stable, we can add it to the master fix location.

Reply:
@WalterKolczynski-NOAA , my preference is to add `fix_jedi` to the Hera and Orion `FIX_DIR` specified in `link_workflow.sh.   
```
elif [ $machine = "hera" ]; then
    FIX_DIR="/scratch1/NCEPDEV/global/glopara/fix_NEW"
elif [ $machine = "orion" ]; then
    FIX_DIR="/work/noaa/global/glopara/fix_NEW"
```